python/raft/dask/common/comms.py

```diff
@@ -0,0 +1,412 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from raft.dask.common.nccl import nccl
```

Comment:
Relative imports

Reply:
Relative imports were missed.

Reply:
Thanks  looking again, Ishika. I've fixed this and gone through to verify the only remaining absolute imports are in the pytests.